webpack/scenes/Subscriptions/Manifest/SimpleContentAccess.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Switch, Level, LevelItem } from '@patternfly/react-core';
+import { OverlayTrigger, Tooltip, Icon } from 'patternfly-react';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { Col, ControlLabel } from 'react-bootstrap';
+
+const SimpleContentAccess = (props) => {
+  const {
+    canToggleSimpleContentAccess,
+    isSimpleContentAccessEnabled,
+    enableSimpleContentAccess,
+    disableSimpleContentAccess,
+    hideModal,
+  } = props;
+
+  const toggleSimpleContentAccess = () => {
+    if (isSimpleContentAccessEnabled) {
+      disableSimpleContentAccess();
+    } else {
+      enableSimpleContentAccess();
+    }
+  };
+
+  const handleChange = () => {
+    toggleSimpleContentAccess();
+    hideModal();
```

Comment:
I don't think it makes sense for this component to be aware of the modal. It might not even be used in the context of a modal. Since the task from toggling is put into the state of the subscriptions page, then the modal should be closed when that task reaches the props.